arbnode/inbox_tracker.go

```diff
@@ -223,6 +228,52 @@ func (t *InboxTracker) GetBatchCount() (uint64, error) {
 	return count, nil
 }
 
+func (t *InboxTracker) FindInboxBatchContainingMessage(pos arbutil.MessageIndex) (uint64, bool, error) {
+	batchCount, err := t.GetBatchCount()
+	if err != nil {
+		return 0, false, err
+	}
+	low := uint64(0)
+	high := batchCount - 1
+	lastBatchMessageCount, err := t.GetBatchMessageCount(high)
+	if err != nil {
+		return 0, false, err
+	}
+	if lastBatchMessageCount <= pos {
+		return 0, false, nil
```

Comment:
I'm not sure this is the right decision, but we could consider returning a special error in this case instead of having the bool return. If we stick with the bool return, we should add a comment explaining what it is.

Reply:
Specialized errors are fickle, especially in interfaces that will later be over rpc. I think a boolean makes sense here.
Added documentation. LMKWYT.

Reply:
Makes sense. I don't see the documentation there, maybe it didn't get pushed up?

Reply:
d'oh!